release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1198,6 +1209,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-scalability-and-performance-bug-fixes_{context}"]
 ==== Scalability and performance
 
+* Previously, you could configure the NUMA resources operator to map a `nodeGroup` to more than one `MachineConfigPool`. This implementation is contrary to the intended design of the Operator, which assumed a one-to-one mapping between a `nodeGroup` and a `MachineConfigPool`. With this release, if a `nodeGroup` maps to more than one `MachineConfigPool`, the Operator accepts the configuration, but the Operator state moves to `Degraded`. To retain the previous behaviour, you can apply the `config.node.openshift-kni.io/multiple-pools-per-tree: enabled` annotation to the NUMA resources operator. However, the ability to assign a `nodeGroup` to more than one `MachineConfigPool` will be removed in a future release. (link:https://issues.redhat.com/browse/OCPBUGS-42523[*OCPBUGS-42523*])
```

Comment:
Please change 'behaviour' to 'behavior.' ([IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=style-overview#basis))